### ISSUE TITLE
can nw.js get the HTTP status code of the page in the window?

### ISSUE THREAD

**Issue by chenxianliang** [nwjs/nw.js#3512]
can nw.js get the HTTP status code of the page in the window?

**Comment by baconface** [comment]
Yes and no. NW.js as far as I know can't. But you can use ```require("http");``` from within an NW.js project to get it. For example, run the url through [http.get](https://nodejs.org/api/http.html#http_http_get_options_callback) to spit out ```res.statusCode```. This is built in so no need to fetch it from NPM.

**Comment by Christywl** [comment]
Please ask questions in our mailing list: https://groups.google.com/forum/#!forum/nwjs-general
